data/newMigrations/003-service_categories.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
```

Comment:
Mistake on my part! I forgot to add that categories have a name too! I updated the schema now.

Just add in a string  that is not nullable for `service_category_name`.

Reply:
Thanks for adding it in!